API.yaml

```diff
@@ -511,36 +572,35 @@ paths:
         required: true
         type : integer
         format: int128
-      - name: tableId
+      - name: providerId
         in: path
-        description: The name of the table to query
+        description: The name of the table provider to query
         required: true
         type: string
-      - name: low
+      - name: lowIndex
         in: query
-        description: Rank of the first event to query
+        description: Index of the first line to query
         required: true
         type: integer
         format: int64
         minimum: 0
       - name: size
         in: query
-        description: Number of events to query
+        description: Number of lines to query
         required: true
         type: integer
         format: int32
-      - name: columns
+      - name: columnId
         in: query
         type: array
         items:
           type: string
-        description: List of column names to return, all columns will be returned if not specified.
+        description: List of column IDs to return, all columns will be returned if not specified.
         required: false
       - name: filters
         in: formData
         type: string
-        description: The form data keys are the column names, their values are
-          the regular expressions to apply to filter the columns
+        description: The form data contains an array of preset filter ID (Key presetfilterId), a boolean to determine of a collapse filter is applied (Key collapseFilter) and a map of column IDs to their regular expressions to apply to filter the columns
```

Comment:
the notions of preset filter and collapse filter are very tc specific, so they might need more explanation before going into the API